src/pn_expressions/mode_weights_Psi_M.jl

```diff
@@ -0,0 +1,198 @@
+            (v/c)^5 * (-4𝒾//3 * √(2//35) * ν)
+        )
+    end
+
+    # ell=5
+    if ℓₘₐₓ ≥ 5
+        Ψ_M[Yindex(5,1,ℓₘᵢₙ)] = Ψ_M₀ * @pn_expansion(
+            # Eq. (A1w) of MSB+ (2022)
+            (v/c)^6 * (52//21*√(2//165)*δ*ν)
+        )
+        Ψ_M[Yindex(5,3,ℓₘᵢₙ)] = Ψ_M₀ * @pn_expansion(
+            # Eq. (A1u) of MSB+ (2022)
+            (v/c)^6 * (4δ*ν/(27√385))
+        )
+        Ψ_M[Yindex(5,5,ℓₘᵢₙ)] = Ψ_M₀ * @pn_expansion(
+            # Eq. (A1s) of MSB+ (2022)
+            (v/c)^6 * (-36δ*ν/(5√77))
+        )
+    end
+
+    # ell=6
+    if ℓₘₐₓ ≥ 6
+        Ψ_M[Yindex(6,0,ℓₘᵢₙ)] = Ψ_M₀ * (-4195/(177408√13)) * @pn_expansion(
+            # Eq. (A1y) of MSB+ (2022)
+            + (v/c)^2 * (1 - 3612ν/839)
+            + (v/c)^4 * (-45661561//6342840 + 101414ν/2517 - 48118ν^2/839)
+            + (v/c)^5 * (1248//839 - 4992ν/839)π
+            + (v/c)^6 * (3012132889099//144921208320
+                         - 27653500031ν/191694720
+                         + 1317967427ν^2/4107744
+                         - 24793657ν^3/342312)
+        )
+    end
+
+    # ell=7
+    # Nothing so far
+
+    # ell=8
+    if ℓₘₐₓ ≥ 8
+        Ψ_M[Yindex(8,0,ℓₘᵢₙ)] = Ψ_M₀ * (75601/(8895744√17)) * @pn_expansion(
+            # Eq. (A1aa) of MSB+ (2022)
+            + (v/c)^4 * (1 - 452070ν/75601 + 733320ν^2/75601)
+            + (v/c)^6 * (-265361599//33869248
+                         + 18177898147ν/321757856
+                         - 722521125ν^2/5745676
+                         + 261283995ν^3/2872838)
+        )
+    end
+
+    # Spin terms
+    if ℓₘₐₓ ≥ 0
+        Ψ_M[Yindex(0,0,ℓₘᵢₙ)] += Ψ_M₀ * @pn_expansion(
+            # Eq. (A1b) of MSB+ (2022)
+            + (v/c)^3 * (14//3 * Sₗ + 2δ * Σₗ) / M^2
+            + (v/c)^4 * ((-16S⃗·S⃗ + 3Σ⃗·Σ⃗ + 32Sₗ^2 + 9Σₗ^2)/12
+                         - 4//3 * δ * (S⃗·Σ⃗ + 2Sₗ*Σₗ)
+                         + 4//3 * ν * (Σ⃗·Σ⃗ + 2Σₗ^2)) / M^4
+        )
+    end
+    if ℓₘₐₓ ≥ 2
+        Ψ_M[Yindex(2,0,ℓₘᵢₙ)] += Ψ_M₀ * 2//7 * √5 * @pn_expansion(
+            # Eq. (A1h) of MSB+ (2022)
+            + (v/c)^3 * (16//3 * Sₗ + 419//160 * δ*Σₗ) / M^2
+            + (v/c)^4 * ((-128S⃗·S⃗ + 24Σ⃗·Σ⃗ + 256Sₗ^2 + 75Σₗ^2) / 96
+                         - 4//3 * δ * (S⃗·Σ⃗ + 2Sₗ*Σₗ)
+                         + 4//3 * ν * (Σ⃗·Σ⃗ + 2Σₗ^2)) / M^4
+        )
+        Ψ_M[Yindex(2,1,ℓₘᵢₙ)] += Ψ_M₀ * 61/(14√30) * @pn_expansion(
+            # Eq. (A1f) of MSB+ (2022)
+            + (v/c)^3 * (- (Sₙ-𝒾*Sλ) - 375//488 * δ * (Σₙ-𝒾*Σλ)) / M^2
+            + (v/c)^4 * (10(3Sₗ*(Sₙ-𝒾*Sλ)+Σₗ*(Σₙ-𝒾*Σλ))
+                         +15δ*((Sₙ-𝒾*Sλ)*Σₗ+Sₗ*(Σₙ-𝒾*Σλ))
+                         -30ν*Σₗ*(Σₙ-𝒾*Σλ))/61M^4
+        )
+    end
+    # ell=3
+    # nothing so far
+    if ℓₘₐₓ ≥ 4
+        Ψ_M[Yindex(4,0,ℓₘᵢₙ)] += Ψ_M₀ * 1//42 * @pn_expansion(
+            # Eq. (A1r) of MSB+ (2022)
+            + (v/c)^3 * (10Sₗ + 57//8 * δ * Σₗ) / M^2
+            + (v/c)^4 * ((-64S⃗·S⃗ + 12Σ⃗·Σ⃗ + 128Sₗ^2 + 41Σₗ^2) / 48
```

Comment:
The minus sign on $64S\cdot S$ is suppose to be acting on the whole expression with $48M^{4}$ in parentheses